### PR TITLE
add nine

### PR DESCRIPTION
**Pull request by looooo**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/nine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - nine
+
+about:
+  home: "https://github.com/nandoflorestan/nine"
+  license: "Public domain"
+  license_family: "PUBLIC-DOMAIN"
+  license_file: ""
+  summary: "Python 2 / 3 compatibility, like six, but favouring Python 3"
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/nine/meta.yaml]: Please remove/fill the unused attributes

File: recipes/nine/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (marcelotrevisani) [inline recipes/nine/meta.yaml]: Please add your github id here

File: recipes/nine/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - nine
+
+about:
+  home: "https://github.com/nandoflorestan/nine"
+  license: "Public domain"
+  license_family: "PUBLIC-DOMAIN"
+  license_file: ""
```
Comment (marcelotrevisani) [inline recipes/nine/meta.yaml]: It should be
```suggestion
  license_file: "LICENSE.rst"
```

**Comment by looooo** [comment]
thanks @marcelotrevisani for reviewing. I somehow added my changes to a wrong file. Should be resolved now.

File: recipes/nine/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3064fbeb512e756a415606a1399f49c22de867d5ac7e2b6c91c35e757d3af42d"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (scopatz) [inline recipes/nine/meta.yaml]: can this be noarch python?

Comment (jakirkham) [reply recipes/nine/meta.yaml]: Probably unless we care about [Python 2.6 support]( https://github.com/nandoflorestan/nine/blob/a50d0cfd8753e2e7944d2a6d7159c5f9cb3fe9ea/setup.py#L13-L16 ), which I'm guessing we don't.

**Comment by jakirkham** [comment]
Thanks for adding this, @looooo! Amazed we didn't already have it. 😄

**Comment by looooo** [comment]
Thanks!

**Pull request merged**